### PR TITLE
NUnit3-console to use AssertionResult collection for Multiple results.

### LINKED ISSUE #127: Modify console runner to display multiple assert information
Currently, the console runner uses the single failure message created for compatibility in order to display the result of a test using multiple asserts. It should use the actual result of each assert to display both the message and stack frame.

### PR DESCRIPTION
**Pull request by CharliePoole**
Fixes #127

### CODE REVIEW

File: src/NUnitConsole/nunit3-console/ResultReporter.cs [hunk 1]
```diff
@@ -260,6 +260,47 @@ private void WriteSummaryCount(string label, int count, ColorStyle color)
...
         private void WriteSingleResult(XmlNode result, ColorStyle colorStyle)
         {
+            string status = GetStatus(result);
+
+            string reportID = (++_reportIndex).ToString();
+            string fullName = result.GetAttribute("fullname");
+            var assertions = result.SelectNodes("assertions/assertion");
+            int numAsserts = assertions.Count;
+
+            //if (numAsserts > 0)
```
Comment (rprouse) [inline src/NUnitConsole/nunit3-console/ResultReporter.cs]: Is this still a Do not merge because of this?

**Comment by CharliePoole** [comment]
It is and also due to some refactoring! I forgot to mark it in the description.

**Comment by CharliePoole** [comment]
Should be ready to merge now. I created a ConsoleTestResult class that ought to be useful in other runners that use the XML format, although the xml code itself will need to be changed. If you look at how it is used, you'll see there is a compromise between efficiency and convenience. Specifically, the class is not used until we have determined that a particular XML node represents a test to be displayed in the report.

File: src/NUnitConsole/nunit3-console/nunit3-console.csproj [hunk 2]
```diff
@@ -45,6 +45,7 @@
     <ApplicationIcon>..\..\..\nunit.ico</ApplicationIcon>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="Microsoft.VisualStudio.QualityTools.UnitTestFramework, Version=10.1.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL" />
```
Comment (rprouse) [inline src/NUnitConsole/nunit3-console/nunit3-console.csproj]: This looks wrong. Isn't this MSTest?

**Comment by rprouse** [comment]
Other than the questionable reference, looks good.

**Comment by rprouse** [comment]
Looks good now. Merging...

**Pull request merged**